bndtools.core/src/bndtools/wizards/bndfile/NewBndFileWizardPage.java

```diff
@@ -57,8 +57,14 @@ protected boolean validatePage() {
         IResource container = ResourcesPlugin.getWorkspace().getRoot().findMember(containerPath);
         IProject project = container.getProject();
 
-        if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
-            error = Messages.NewBndFileWizardPage_errorReservedFilename;
+        try {
+            if (project.hasNature(BndtoolsConstants.NATURE_ID)) {
+                if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
+                    error = Messages.NewBndFileWizardPage_errorReservedFilename;
```

Comment:
here is the current warning in RC2

![screen shot 2016-09-07 at 12 45 00 pm](https://cloud.githubusercontent.com/assets/595221/18322574/f8e80e92-74f8-11e6-8289-c6ebfec85741.png)